mocha.css

```diff
@@ -292,11 +292,12 @@ body {
   -moz-transition:opacity 200ms;
   -o-transition:opacity 200ms;
   transition: opacity 200ms;
-  opacity: 0.3;
+  opacity: 0.7;
   color: var(--mocha-test-replay-color);
 }
 
 #mocha .test:hover a.replay {
+  box-shadow: 0 0 1px inset var(--mocha-test-replay-bg-color);
```

Comment:
@JoshuaKGoldberg thank you. One more question, I'm sorry.

Does this line have any effect, since the box-shadow has the same color as the background? Just the opacity changes from 0.7 to 1?

Reply:
Ah good point, I'd meant to switch this to having a slight outline color: `--mocha-test-bg-color`. Thanks!